operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -14,6 +10,11 @@ powerscale:
       replication: "v1.5.0"
       observability: "v1.5.0"
       resiliency: "v1.6.0"
+  v2.8.0:
+      authorization: "v1.7.0"
+      replication: "v1.5.0"
+      observability: "v1.5.0"
+      resiliency: "v1.6.0"
```

Comment:
resiliency: "v1.6.0" -> resiliency: "v1.7.0" ? Correct the version here too, like CSI-PowerStore.

Reply:
Done